src/VariableInput/VariableInput.st.css

```
@@ -80,19 +78,17 @@
 .root .tagWrapper {
   padding-left: 2px;
   padding-right: 2px;
-  vertical-align: middle;
 }
 .root .textWrapper {
   -st-mixin: text-medium-normal;
   display: inline;
-  vertical-align: middle;
-}
-.root:size(large) .textWrapper {
-  line-height: 30px;
 }
 .root:size(small) .textWrapper {
   -st-mixin: text-small-normal;
 }
+.root :global(.public-DraftEditor-content > div) {
```

I would do the following instead of relying on an internal direct child:
```suggestion
.root :global(.public-DraftStyleDefault-block) {
```